src/api/java/mekanism/api/MekanismAPITags.java

```diff
@@ -86,6 +86,10 @@ private Chemicals() {
          * Represents all clean slurries.
          */
         public static final TagKey<Chemical> CLEAN = tag("clean");
+        /**
+         * Chemicals in this tag cannot be inserted into framed blocks
+         */
```

Comment:
This should have a since 10.7.1 tag and please also edit the api version in MekanismAPI to be 10.7.1